src/v2/stores/nodes.js

```diff
@@ -58,7 +58,7 @@ class Store {
           pubkey,
           gossip,
           coordinates,
-          name: (identity && identity.name) || identity.pubkey,
+          name: (identity && (identity.name || identity.pubkey)) || '',
```

Comment:
I think it should be:

`(identity && identity.name) || pubkey || ''`

Reply:
if you insist, both are ok. identity has pubkey field